release/release.go

```diff
@@ -159,20 +167,20 @@ func buildDeb() {
 
 	// create DEBIAN dir
 	controlDir := filepath.Join(releaseName, "DEBIAN")
-	check(os.MkdirAll(controlDir, os.ModePerm), "mkdirAll " + controlDir)
+	check(os.MkdirAll(controlDir, os.ModePerm), "mkdirAll "+controlDir)
 
 	// create usr/bin and usr/share/doc
 	binDir := filepath.Join(releaseName, "usr", "bin")
-	check(os.MkdirAll(binDir, os.ModePerm), "mkdirAll " + binDir)
+	check(os.MkdirAll(binDir, os.ModePerm), "mkdirAll "+binDir)
 	docDir := filepath.Join(releaseName, "usr", "share", "doc", mdt)
-	check(os.MkdirAll(docDir, os.ModePerm), "mkdirAll " + docDir)
+	check(os.MkdirAll(docDir, os.ModePerm), "mkdirAll "+docDir)
 
 	md5sums := make(map[string]string)
 	// We use the order just to make sure the md5sums are always in the same order.
 	// This probably doesn't matter, but it looks nicer for anyone inspecting the md5sums file.
-	md5sumsOrder := make([]string, 0, len(binaries) + len(staticFiles))
+	md5sumsOrder := make([]string, 0, len(binaries)+len(staticFiles))
```

Comment:
`gofmt` did this

Reply:
We should probably set gofmt to run automatically at some point. But I've always felt that way :D